src/main/java/ru/hogwarts/school/controller/FacultyController.java

```diff
@@ -0,0 +1,62 @@
+package ru.hogwarts.school.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ru.hogwarts.school.model.Faculty;
+import ru.hogwarts.school.service.FacultyService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/faculty")
+public class FacultyController {
+    private FacultyService facultyService;
+
+    public FacultyController(FacultyService facultyService) {
+        this.facultyService = facultyService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Faculty> createFaculty(@RequestBody Faculty faculty) {
+        return ResponseEntity.ok(faculty);
+    }
```

Comment:
Не хватает вызова метода сохранения
```suggestion
    @PostMapping
    public ResponseEntity<Faculty> createFaculty(@RequestBody Faculty faculty) {
        return ResponseEntity.ok(facultyService.createFaculty(faculty));
    }
```